src/ImageSharp.Formats/ConfigurationExtensions.cs

```diff
@@ -0,0 +1,32 @@
+﻿// <copyright file="ConfigurationExtensions.cs" company="James Jackson-South">
+// Copyright (c) James Jackson-South and contributors.
+// Licensed under the Apache License, Version 2.0.
+// </copyright>
+
+namespace ImageSharp
+{
+    using System;
+    using System.IO;
+
+    using Formats;
+
+    /// <summary>
+    /// Extension methods for the <see cref="Configuration"/> type.
+    /// </summary>
+    public static partial class ConfigurationExtensions
+    {
+        /// <summary>
+        /// Adds the common formats, PNG, JPEG, GIF and BMP.
+        /// </summary>
+        /// <param name="config">The config.</param>
+        /// <returns>The config</returns>
+        public static Configuration AddCommonFormats(this Configuration config)
```

Comment:
It's very practical to have this method, but I don't like the idea of having a whole package for this, messing up the dependency graph.
I'd rather define a simple, convention based utility which is placed in `ImageSharp.dll`, and uses reflection without enumerating / loading all the dll-s, only looks for ImageSharp core modules.